File: INICIANTE/1064/main.dart
import 'dart:io';

void formattedPrint(int positives, double values) {
  print('$positives valores positivos');
  print((values / positives).toStringAsFixed(1));
}

void main() {
  int positives = 0;
  double value, values = 0;
  for (int times = 0; times < 6; times++) {
    if ((value = double.parse(stdin.readLineSync()!)) > 0) {
      positives++;
      values += value;
    }
  }
  formattedPrint(positives, values);
}

File: INICIANTE/1064/main.py
def formatted_print(positives, values):
    print(f'{positives} valores positivos')
    print(f'{values / positives:.1f}')


positives = 0
values = 0
for _ in range(6):
    if (value := float(input())) > 0:
        positives += 1
        values += value
formatted_print(positives, values)
